### ISSUE TITLE
Add a Visitor Counter

### ISSUE THREAD

**Issue by Peart-Guy** [kvcops/KV-Nexus#55]
I want to add add a visitor counter to your web app

@kvcops  Kindly assign me this issue

![Screenshot 2024-10-26 004024](https://github.com/user-attachments/assets/3611fc77-652f-4e51-bb75-c385cd6a9fcf)
![Screenshot 2024-10-26 004029](https://github.com/user-attachments/assets/4fd2d297-1e33-4027-b89c-db0a5d1b47a9)

**Comment by kvcops** [comment]
Can you explain about the use cases for adding the visitors option

**Comment by Peart-Guy** [comment]
> Can you explain about the use cases for adding the visitors option

It will show the visitor count of the web app

**Comment by kvcops** [comment]
Yep...but this platform is mostly designed for all of the audience . I think this feature just takes up the space in the page ...it seems like it might not be helpful ...do you have any suggestions

**Comment by Peart-Guy** [comment]
> Yep...but this platform is mostly designed for all of the audience . I think this feature just takes up the space in the page ...it seems like it might not be helpful ...do you have any suggestions

Well it is completely upto you

**Comment by Wahid7852** [comment]
Closing this. We don't have any plans for this
